integration/server_test.go

```diff
@@ -318,7 +318,7 @@ func TestClientTLS(t *testing.T) {
 	// Assert regular client receives error
 	_, err = testServerClient().Get(fmt.Sprintf("https://localhost:%d/example", port))
 	require.Error(t, err, "Client allowed to make request without cert")
-	assert.Contains(t, err.Error(), "tls: bad certificate")
+	assert.ErrorContains(t, err, "tls", "expected error to contain 'tls' in its string representation")
```

Comment:
The error string here changed between 1.20 and 1.21. To accommodate I updated the test to check look for tls somewhere in the error string.

In go 1.20 the error is `tls: bad certificate`. In go 1.21 it's `tls: certificate required`.